Example/Backpack/SwiftUI/Components/Rating/RatingExampleViewWithTitle.swift

```diff
@@ -0,0 +1,61 @@
+//
```

Comment:
Can we remove this extra //

Reply:
updated in  27fa2c175db76551ee1eeae2b1f720dd65151395